aplib.net-demo/Assets/Scripts/Entities/EntitySound.cs

```diff
@@ -18,17 +17,12 @@ public class EntitySound : MonoBehaviour
 
     private AudioSource _audioSource;
 
-    private HealthComponent _healthComponent;
-
     /// <summary>
     /// Sets up the AudioSource component.
     /// </summary>
     private void Awake()
     {
         _audioSource = GetComponent<AudioSource>();
-        _healthComponent = GetComponent<HealthComponent>();
-
-        _healthComponent.Death += OnDeath;
     }
```

Comment:
Why did you remove this? Subscribing a Death sound to the health component was the better way to do this, this seems like a wrong merge again?